odom:
  marg: false
  num_kfs: 4
  num_levels: 5
  min_track_ratio: 0.6
  vis_min_depth: 1.0
  reinit: false
  init_depth: false
  init_stereo: true
  init_align: true
select:
  sel_level: 1
  cell_size: 16
  min_grad: 8
  max_grad: 64
  nms_size: 1
  min_ratio: 0.4
  max_ratio: 0.6
  reselect: true
stereo:
  half_rows: 2
  half_cols: 3
  match_level: 3
  refine_size: 1
  min_zncc: 0.8
  min_depth: 1.0
align:
  init_level: 0
  max_iters: 8
  max_xs: 0.1
  affine: false
  stereo: false
  c2: 2
  dof: 4
  max_outliers: 1
  grad_factor: 1.5
  min_depth: 0.1
adjust:
  init_level: 2
  max_iters: 5
  max_xs: 0.1
  affine: false
  stereo: true
  c2: 2
  dof: 4
  max_outliers: 0
  grad_factor: 1.5
  min_depth: 0.1
